Description: do not prefix the configuration path with the INSTALL_PREFIX.
 In the 3.3.0 branch, INSTALL_PREFIX (in our case, /usr) was prepended to 
 the config file path, thus breaking any standard and using the ugly 
 /usr/etc/hhvm dir. This also breaks our package as we install configs in 
 /etc/hhvm/.

Author: Giuseppe Lavagetto <glavagetto@wikimedia.org>
Last-Updated: 2014-09-26

diff --git a/hphp/runtime/base/emulate-zend.cpp b/hphp/runtime/base/emulate-zend.cpp
index 52636af..f2af4f1 100644
--- a/hphp/runtime/base/emulate-zend.cpp
+++ b/hphp/runtime/base/emulate-zend.cpp
@@ -229,12 +229,12 @@ int emulate_zend(int argc, char** argv) {
 
     // If the -c option is specified without a -n, php behavior is to
     // load the default ini/hdf
-    auto default_config_file = INSTALL_PREFIX "/etc/hhvm/php.ini";
+    auto default_config_file = "/etc/hhvm/php.ini";
     if (access(default_config_file, R_OK) != -1) {
       newargv.push_back("-c");
       newargv.push_back(default_config_file);
     }
-    default_config_file = INSTALL_PREFIX "/etc/hhvm/config.hdf";
+    default_config_file = "/etc/hhvm/config.hdf";
     if (access(default_config_file, R_OK) != -1) {
       newargv.push_back("-c");
       newargv.push_back(default_config_file);
